omop_file_validator.py

```diff
@@ -359,21 +360,22 @@ def run_checks(file_path, f):
                                 submission_column, meta_column_type,
                                 submission_column_type, df)
                             if error_row_index:
-                                e = dict(message=MSG_INVALID_TYPE +
-                                         " line number " +
-                                         str(error_row_index + 1),
-                                         column_name=submission_column,
-                                         actual=df[submission_column]
-                                         [error_row_index],
-                                         expected=meta_column_type)
-                                result['errors'].append(e)
+                                # convert pd nan values to empty string
+                                df = df.replace(np.nan, '')
+                                if df[submission_column][error_row_index] == '' and not meta_column_required:
+                                    continue
+                                else:
```

Comment:
This can probably be simplified to
```suggestion
                                if not (df[submission_column][error_row_index] == '' and not meta_column_required):
```

Reply:
Ah good catch, thank you!